python/fate_flow/pipelined_model/publish_model.py

```diff
@@ -58,7 +58,9 @@ def load_model(config_data):
             load_model_request.local.role = config_data.get('local').get('role')
             load_model_request.local.partyId = config_data.get('local').get('party_id')
             load_model_request.loadType = config_data['job_parameters'].get("load_type", "FATEFLOW")
-            if not get_base_config('use_registry'):
+            # make use of 'model.transfer.url' in serving server
+            use_serving_url = config_data['job_parameters'].get('use_serving_transfer_url', False)
```

Comment:
Consider modifying "use_serving_transfer_url" to "use_transfer_url_on_serving"?

Reply:
SURE